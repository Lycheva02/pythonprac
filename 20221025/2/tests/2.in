print(*list(slide('TRY TO READ THIS TEXT', 4)))