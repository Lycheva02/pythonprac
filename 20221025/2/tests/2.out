T R Y   R Y   T Y   T O   T O   T O   R O   R E   R E A R E A D E A D   A D   T D   T H   T H I T H I S H I S   I S   T S   T E   T E X T E X T E X T X T T
